core/domain/fs_services.py

```diff
@@ -143,10 +370,9 @@ def copy_images(
         filenames: list(str). The list of filenames to copy.
     """
     file_system_class = get_entity_file_system_class()
```

Comment:
Can we also get rid of `get_entity_file_system_class`? It only returns one thing and that seems a bit useless to me.

Reply:
Done.

Reply:
Done.